### PR TITLE
[BUILD] Enforce package naming conventions PMD

### PR DESCRIPTION
**Pull request by m273d15**
* Add PackageCase rule
* Fix package naming

### CODE REVIEW

**Comment by saros-infrastructure** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- eclipse/src/saros/ui/preference_pages/FeedbackPreferencePage.java  2
- eclipse/src/saros/ui/command_handlers/SessionAddContactsHandler.java  1
- eclipse/src/saros/ui/preference_pages/CommunicationPreferencePage.java  3
- eclipse/src/saros/ui/command_handlers/SessionAddProjectsHandler.java  1
- eclipse/src/saros/ui/command_handlers/SessionAddSelectedContactsHandler.java  1
- eclipse/src/saros/ui/preference_pages/AdvancedPreferencePage.java  2
- eclipse/src/saros/ui/preference_pages/PersonalizationPreferencePage.java  2
- eclipse/src/saros/ui/menu_contributions/StartSessionWithProjects.java  4
- eclipse/src/saros/ui/preference_pages/NetworkPreferencePage.java  12
- eclipse/src/saros/ui/command_handlers/SessionAddSelectedProjectsHandler.java  1
- eclipse/src/saros/ui/command_handlers/AddContactHandler.java  1
- eclipse/src/saros/ui/command_handlers/AddXMPPAccountHandler.java  1
- eclipse/src/saros/ui/preference_pages/GeneralPreferencePage.java  2
- eclipse/src/saros/ui/command_handlers/SessionLeaveHandler.java  1
- eclipse/src/saros/ui/command_handlers/StartSarosConfigurationHandler.java  1
- eclipse/src/saros/ui/command_handlers/CreateXMPPAccountHandler.java  1
- eclipse/src/saros/service_providers/SarosSourceProvider.java  2
- eclipse/src/saros/ui/menu_contributions/StartSessionWithContacts.java  5
- eclipse/src/saros/ui/command_handlers/GettingStartedHandler.java  1
- eclipse/src/saros/ui/command_handlers/ShareProjectHandler.java  1
- eclipse/src/saros/ui/command_handlers/OpenSarosPreferencesHandler.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Saros/saros/pullRequest?prid=5586006&bid=18306723)

**Comment by saros-infrastructure** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- eclipse/src/saros/ui/preference_pages/CommunicationPreferencePage.java  3
- eclipse/src/saros/ui/preference_pages/PersonalizationPreferencePage.java  2
- eclipse/src/saros/ui/menu_contributions/StartSessionWithContacts.java  5
- eclipse/src/saros/ui/menu_contributions/StartSessionWithProjects.java  4
- eclipse/src/saros/ui/preference_pages/FeedbackPreferencePage.java  2
- eclipse/src/saros/ui/preference_pages/GeneralPreferencePage.java  2
- eclipse/src/saros/ui/command_handlers/SessionAddContactsHandler.java  1
- eclipse/src/saros/ui/command_handlers/GettingStartedHandler.java  1
- eclipse/src/saros/ui/command_handlers/OpenSarosPreferencesHandler.java  2
- eclipse/src/saros/ui/command_handlers/StartSarosConfigurationHandler.java  1
- eclipse/src/saros/ui/command_handlers/ShareProjectHandler.java  1
- eclipse/src/saros/ui/command_handlers/SessionLeaveHandler.java  1
- eclipse/src/saros/ui/command_handlers/AddContactHandler.java  1
- eclipse/src/saros/ui/command_handlers/SessionAddSelectedProjectsHandler.java  1
- eclipse/src/saros/ui/command_handlers/SessionAddSelectedContactsHandler.java  1
- eclipse/src/saros/ui/preference_pages/NetworkPreferencePage.java  12
- eclipse/src/saros/ui/command_handlers/CreateXMPPAccountHandler.java  1
- eclipse/src/saros/ui/command_handlers/AddXMPPAccountHandler.java  1
- eclipse/src/saros/ui/command_handlers/SessionAddProjectsHandler.java  1
- eclipse/src/saros/ui/preference_pages/AdvancedPreferencePage.java  2
- eclipse/src/saros/service_providers/SarosSourceProvider.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Saros/saros/pullRequest?prid=5586006&bid=18306723)

File: ruleset.xml [hunk 1]
```diff
@@ -10,4 +10,6 @@
     <rule ref="category/java/bestpractices.xml/UnusedLocalVariable" />
     <rule ref="category/java/bestpractices.xml/UnusedPrivateField" />
     <rule ref="category/java/bestpractices.xml/UnusedPrivateMethod" />
+
+    <rule ref="category/java/codestyle.xml/PackageCase" />
```
Comment (tobous) [inline ruleset.xml:14]: Is there a specific reason to separate this rule from the others?

Comment (m273d15) [reply ruleset.xml:14]: The rule is part of the "codestyle" pmd ruleset (and the other rules are part of the "bestpractices" set). I just group by the rules by their origin.

**Comment by saros-infrastructure** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- eclipse/src/saros/ui/preference_pages/AdvancedPreferencePage.java  2
- eclipse/src/saros/ui/command_handlers/SessionAddContactsHandler.java  1
- eclipse/src/saros/ui/command_handlers/StartSarosConfigurationHandler.java  1
- eclipse/src/saros/ui/command_handlers/SessionAddSelectedProjectsHandler.java  1
- eclipse/src/saros/ui/command_handlers/CreateXMPPAccountHandler.java  1
- eclipse/src/saros/service_providers/SarosSourceProvider.java  2
- eclipse/src/saros/ui/menu_contributions/StartSessionWithProjects.java  4
- eclipse/src/saros/ui/menu_contributions/StartSessionWithContacts.java  5
- eclipse/src/saros/ui/preference_pages/PersonalizationPreferencePage.java  2
- eclipse/src/saros/ui/command_handlers/ShareProjectHandler.java  1
- eclipse/src/saros/ui/command_handlers/SessionAddProjectsHandler.java  1
- eclipse/src/saros/ui/command_handlers/AddXMPPAccountHandler.java  1
- eclipse/src/saros/ui/preference_pages/CommunicationPreferencePage.java  3
- eclipse/src/saros/ui/command_handlers/SessionAddSelectedContactsHandler.java  1
- eclipse/src/saros/ui/preference_pages/FeedbackPreferencePage.java  2
- eclipse/src/saros/ui/command_handlers/GettingStartedHandler.java  1
- eclipse/src/saros/ui/preference_pages/GeneralPreferencePage.java  2
- eclipse/src/saros/ui/preference_pages/NetworkPreferencePage.java  12
- eclipse/src/saros/ui/command_handlers/SessionLeaveHandler.java  1
- eclipse/src/saros/ui/command_handlers/AddContactHandler.java  1
- eclipse/src/saros/ui/command_handlers/OpenSarosPreferencesHandler.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Saros/saros/pullRequest?prid=5586006&bid=18306723)

**Pull request merged**